net.bartkessels.getit.json

```diff
@@ -33,6 +33,20 @@
 				}
 			]
 		},
+		{
+			"name": "libsoup-2.4",
+			"buildsystem": "meson",
+			"config-opts": [
+				"-Dtests=false"
+			],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://download.gnome.org/sources/libsoup/2.74/libsoup-2.74.3.tar.xz",
+					"sha256": "e4b77c41cfc4c8c5a035fcdc320c7bc6cfb75ef7c5a034153df1413fa1d92f13"
+				}
+			]
+		},
```

Comment:
Please use this from shared modules https://github.com/flathub/shared-modules/tree/master/libsoup